### ISSUE TITLE
App Canary: fix accessibility issues

### ISSUE THREAD

**Issue by cgraham-rs** [posit-dev/connect-extensions#198]
There are some accessibility failures due to contrast, and when added, the log and email icons are small and difficult to see.
